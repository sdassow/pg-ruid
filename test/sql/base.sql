
\set ECHO 0

BEGIN;

\i sql/ruid.sql

SELECT 'AAAAAAAAAAAAAAAAAAAAAA'::ruid;

ROLLBACK;

