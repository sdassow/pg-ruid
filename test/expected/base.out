\set ECHO 0
          ruid          
------------------------
 AAAAAAAAAAAAAAAAAAAAAA
(1 row)

